030-Concepts/075-file-attachments.mdx

```diff
@@ -190,3 +190,60 @@ img = xata.files().transform("https://us-east-1.storage.xata.sh/4u1fh2o6p10blbut
 ````
 
 </TabbedCode>
+
+### Upload URLs
+
+An upload URL offers authenticated access to upload/update a file without requiring an API key. The URL contains an expiring signature allowing anyone holding the URL to upload a file during the configured duration. Similar to signed URLs, they need to be requested directly and do not come along with the default request on the record.
+
+Http `PUT` is the only valid method for a request using the upload URL.
+
+Note that an upload URL targets a specific file, therefore multiple uploads using the same URL will replace the content of the same file rather than creating multiple files.
```

Comment:
```suggestion
An upload URL is assigned to a specific file. Subsequent uploads using the same URL will overwrite the existing file content, instead of generating additional files. Every time the URL is used, it updates the same specific file it's linked to.
```